web/web.go

```diff
@@ -145,14 +145,8 @@ func root(c *api.Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(c.Session.UserId) == 0 {
```

Comment:
So you plan on restoring this feature in another way? What's wrong with redirects?